mysite/settings.py

```diff
@@ -10,19 +10,22 @@
 https://docs.djangoproject.com/en/2.0/ref/settings/
 """
 
-import os
+import environ
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
-BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+BASE_DIR = environ.Path(__file__) - 2  # 2 folder back
+env = environ.Env(DEBUG=(bool, False), )  # pylint: disable=invalid-name
+environ.Env.read_env()  # reading .env file
```

Comment:
I'd call this method from the `env` var. Is it possible?

Reply:
@webknjaz Sorry, do you mean not `environ.Env.read_env()` but I need `env.read_env()`, right?